### DISCUSSION TITLE
🚀 Feature request for ExpressionEngine.com License management

### DISCUSSION THREAD

**Discussion by creativearc-paul** [ExpressionEngine/ExpressionEngine#4531] [Feature Requests]
### Suggest an idea for ExpressionEngine

This one is pretty simple I hope.

While reviewing the list of Sites (/store/licenses > Manage Sites), all the domains are listed along with license number.

BUT, when you navigate to an individual site, the license is not listed within the individual Site overview. I often find myself having to awkwardly navigate back to retrieve the license.

Could we repeat/include the license # on the individual Site page?

<img width="1289" alt="image" src="https://github.com/user-attachments/assets/fa6fe57c-637d-4196-9d8b-a8e094816b51">

<img width="1116" alt="image" src="https://github.com/user-attachments/assets/770e334d-67c6-4235-b793-5027d0daf64d">

**Comment by creativearc-paul** [comment]
I may be missing something obvious, but I just don't see it!

**Comment by intoeetive** [comment]
@TomJaeger @bryannielsen @matthewjohns0n @robinsowell please consider

**Comment by dougblackjr** [comment]
+1 for this one. Would be nice to be able to see sites at a glance (with all of their associated licenses)!
